make/mt-config/mt-config.mk

```diff
@@ -78,6 +78,8 @@ XI_CONFIG_FLAGS += -DXI_DEBUG_OUTPUT=$(XI_DEBUG_OUTPUT)
 XI_CONFIG_FLAGS += -DXI_DEBUG_ASSERT=$(XI_DEBUG_ASSERT)
 XI_CONFIG_FLAGS += -DXI_DEBUG_EXTRA_INFO=$(XI_DEBUG_EXTRA_INFO)
 
+XI_CONFIG_FLAGS += -DUSE_CBOR_CONTEXT
```

Comment:
Enabling CBOR context enables us to inject memory handler functions (alloc, calloc). Thus this helps us to create leak free code.